backend/private-graph/graph/resolver_test.go

```diff
@@ -376,6 +379,14 @@ func TestResolver_GetAIQuerySuggestion(t *testing.T) {
 			productType: modelInputs.ProductTypeLogs,
 			query:       "all logs with level error, between 4pm yesterday and just now.",
 		},
+		"irrelevant query": {
+			productType: modelInputs.ProductTypeLogs,
+			query:       openai_interface.IrrelevantQuery,
```

Comment:
i'd move the `IrrelevantQuery` variable to the test, since it's only relevant to the test